RELEASE_LOG.md

```diff
@@ -137,3 +137,6 @@
 
 ## Tickets
 * i979: Push docker images to docker hub
+
+## Other branches merged in this release
+
```

Comment:
just fixing our previously added manual edit to the release log, for consistency